app/services/classification/base.rb

```diff
@@ -160,6 +160,12 @@ def get_element_and_element_name(lookup_value)
       element_name = ''
       lookup_value.match.split(LookupKey::KEY_DELM).each do |match_key|
         lv_element, lv_element_name = match_key.split(LookupKey::EQ_DELM)
+        case lv_element
```

Comment:
I'm a little unsure about this because this is actually the matcher defined by the user in the Puppet class settings.  To change it for some objects and not others might be confusing, and open up another can of worms.